Makefile

```diff
@@ -57,6 +57,7 @@ bins:
 	@which bin/br
 	@which bin/tiflash
 	@which bin/libtiflash_proxy.so
+	@which bin/cdc
```

Comment:
We could update tests/readme.md at the same time.

Reply:
sure